examples/immersive_paris.html

```diff
@@ -0,0 +1,172 @@
+                var layer = new itowns.ColorLayer('Ortho', config);
+                view.addLayer(layer).then(menuGlobe.addLayerGUI.bind(menuGlobe));
+            });
+
+            // Add two elevation layers.
+            // These will deform iTowns globe geometry to represent terrain elevation.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                var layer = new itowns.ElevationLayer(config.id, config);
+                view.addLayer(layer).then(menuGlobe.addLayerGUI.bind(menuGlobe));
+            }
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+            function colorBuildings(properties) {
+                if (properties.id.indexOf('bati_remarquable') === 0) {
+                    return new itowns.THREE.Color(0x5555ff);
+                } else if (properties.id.indexOf('bati_industriel') === 0) {
+                    return new itowns.THREE.Color(0xff5555);
+                }
+                return new itowns.THREE.Color(0xeeeeee);
+            }
+
+            function altitudeBuildings(properties) {
+                return properties.z_min - properties.hauteur - 3;
+            }
+
+            function extrudeBuildings(properties) {
+                return properties.hauteur + 3;
+            }
+
+            // Prepare oriented image source
+            var orientedImageSource = new itowns.OrientedImageSource({
+                    url: 'http://www.itowns-project.org/itowns-sample-data-small/images/140616/Paris-140616_0740-{cameraId}-00001_0000{panoId}.jpg'
+                });
+
+            // Url to a GEOJSON file describing feature points. It describre position and orientation of each panoramic.
+            var orientationsUrl = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/panoramicsMetaDataParis.geojson';
+            // Url of a a JSON file with calibration for all cameras. see [CameraCalibrationParser]{@link module:CameraCalibrationParser.parse}
+            // in this example, we have the ladybug, it's a set of 6 cameras
+            var calibrationUrl = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/cameraCalibration.json';
+            
+            // Fetch the two files
+            var promises = [];
+            var networkOptions = {crossOrigin: ''};
+            promises.push(itowns.Fetcher.json(orientationsUrl, networkOptions) );
+            promises.push(itowns.Fetcher.json(calibrationUrl, networkOptions) ) ;
+            Promise.all(promises).then((res) => {
+
+                var orientation = res[0];
+                var calibration = res[1];
+
+                // Create oriented image layer
+                var olayer = new itowns.OrientedImageLayer('demo_orientedImage', {
+                    sphereRadius: 50,
```

Comment:
`sphereRadius` it's to backgroung? Could you add default value? and remove in example?

Reply:
if it's exposed, it should be documented